File: Quan-Ly-Phong-Gym/LoaiSP.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Quan_Ly_Phong_Gym
{
    public class LoaiSP
    {
        public int maLoai { get; set; }
        public string tenLoai { get; set; }

        public LoaiSP(int MaLoai, string TenLoai)
        {
            this.maLoai = MaLoai;
            this.tenLoai = TenLoai;
        }

        public LoaiSP(LoaiSP Loai)
        {
            tenLoai = Loai.tenLoai;
            maLoai = Loai.maLoai;
        }
    }
}

File: qlgym/src/main/java/mypack/LoaiSP.java
package mypack;

public class LoaiSP {
    public int maLoai;
    public String tenLoai;

    public LoaiSP(int MaLoai, String TenLoai) {
        this.maLoai = MaLoai;
        this.tenLoai = TenLoai;
    }

    public LoaiSP(LoaiSP Loai) {
        tenLoai = Loai.tenLoai;
        maLoai = Loai.maLoai;
    }

    public int getMaLoai() {
        return this.maLoai;
    }

    public void setMaLoai(int maLoai) {
        this.maLoai = maLoai;
    }

    public String getTenLoai() {
        return this.tenLoai;
    }

    public void setTenLoai(String tenLoai) {
        this.tenLoai = tenLoai;
    }

}
